### PR TITLE
Covidx6 script

### PR DESCRIPTION
**Pull request by NaomiTerhljan**
# Pull Request Template

## Description

Creates a covidx6 .ipynb for generating train and test datasets for binary classification. I've also updated README documentation on the main page and dataset page.

Alex's message from slack with the requirements for the script is as follows:

Create a new v6 data curation script that generates a new training dataset such that: 1) we now include the ActuaMed covid-negative data as well as the Cohen covid-negative data (i.e., anything that isn't COVID-positive) alongside the RSNA data under COVID-19 negative, 2) have just two labels (covid-negative and covid-positive), 3) have the test set containing the same 100 covid-positive cases but have the negative test set containing a random selection of 10 from the existing normal cases in current test set, 70 from the existing pneumonia cases in current test set, and random selection of 20 negative cases from Actuamed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 **Recording to webinar on [How we built COVID-Net in 7 days with Gensynth](https://darwinai.news/fny)**
 
+**Update 12/23/2020:** Released a new COVIDx6 dataset for binary classification (COVID-19 positive or COVID-19 negative) with over 14500 CXR images.\
```
Comment (haydengunraj) [inline README.md:7]: If any other changes to the PR are required, change this to "We released..." to match the previous update messages. Otherwise, I don't think it's worth adding a new commit just for this change.

File: test_COVIDx6.txt [hunk 2]
```diff
@@ -0,0 +1,200 @@
+ANON193 CR.1.2.840.113564.1722810170.20200418154808640220.1003000225002.png negative actmed
```
Comment (haydengunraj) [inline test_COVIDx6.txt:1]: In general, I think we should start putting the label files into their own directory rather than the top level of the repo. As we release more versions, this would help keep the top level of the repo less cluttered. @mayaliliya @NaomiTerhljan what are your thoughts on this?

File: create_COVIDx_binary.ipynb [hunk 3]
```diff
@@ -0,0 +1,500 @@
+{
```
Comment (haydengunraj) [inline create_COVIDx_binary.ipynb:1]: I'll try to run this to double-check the numbers when I have a chance, but don't let that hold you up if you want to merge before then.

File: test_COVIDx6.txt (hunk 2 above)
Comment (mayaliliya) [reply test_COVIDx6.txt:1]: I agree, that might be nice moving forward. We can either do this in this PR or another.
Also, curious for thoughts whether we should just make it one test_COVIDx file, as they are all the same. Thoughts?

Comment (NaomiTerhljan) [reply test_COVIDx6.txt:1]: @haydengunraj I think that's a good idea I can do that in this PR! And @mayaliliya maybe we should keep it for now because this test file is different than the past ones.... As well the new dataset script randomly samples the 'negative' test files from the available test files from each source so the test may change from here on out (this was done based on Alex's description but maybe next meeting we can discuss if we want to keep this behavior or have it constant)

Comment (NaomiTerhljan) [reply test_COVIDx6.txt]: @haydengunraj I have created a new 'labels' folder now and updated the train_tf.py default paths accordingly

**Review by mayaliliya — APPROVED** [review]
